plugins/generic/acron/locale/fa_IR/locale.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../dtd/locale.dtd">
+<!--
+  * locale.xml
+  *
+  * Copyright (c) 2013-2017 Simon Fraser University
+  * Copyright (c) 2000-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the en_US (U.S. English) locale.
```

Comment:
Update the text here for the right locale name (or better yet, remove it -- I've just done that from [the English text](https://github.com/pkp/pkp-lib/commit/21623a3406ecdbc2559913a58ef8243514bd0864), as it's just extra maintenance without adding value.)